data/json/vehicleparts/combustion.json

```
@@ -31,6 +31,7 @@
     "copy-from": "engine_combustion",
     "type": "vehicle_part",
     "fuel_type": "gasoline",
+    "fuel_options": [ "gasoline", "avgas" ],
```

```suggestion
    "fuel_options": [ "gasoline", "avgas" ],
    "//2": "TODO: remove when injectors are implemented",
```
could you also add some note that this is sort of tech debt?